community-guidelines.html

```diff
@@ -0,0 +1,57 @@
+</head>
+<body>
+    <header>        
+        <p><img src="https://i.imgur.com/88pvVBP.png" height="200"></p>
+    </header>
+
+    <main>
+        
+<h1>Community Guidelines</h1>
+<p>At Aerobridge, we recognize and celebrate the creativity and collaboration of open source contributors and the diversity of skills, experiences, cultures, and opinions they bring to the projects and communities they participate in.</p>
+<p>The Aerobridge's open-source project and community is inclusive environments, based on treating all individuals respectfully, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristic.</p>
+
+
+<p>We value diverse opinions, but we value respectful behaviour more.<br>
+    Respectful behaviour includes:
+    </p>
+<ul>
+<li>Being considerate, kind, constructive, and helpful.</li>
+<li>Not engaging in demeaning, discriminatory, harassing, hateful, sexualized, or physically threatening behaviour, speech, and imagery.
+</li>
+<li>Not engaging in unwanted physical contact.
+</li>
+
+</ul>
+Aerobridge has also adopted a <a href="https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/">modified Contributor Covenant.</a>
+<h2>Resolve peacefully</h2>
+
+<p>We do not believe that all conflict is necessarily bad; healthy debate and disagreement often yields positive results. However, it is never okay to be disrespectful.
+   <br> If you see someone behaving disrespectfully, you are encouraged to address the behaviour directly with those involved. Many issues can be resolved quickly and easily, and this gives people more control over the outcome of their dispute. If you are unable to resolve the matter for any reason, or if the behaviour is threatening or harassing, report it. We are dedicated to providing an environment where participants feel welcome and safe.
+    </p>
+
+<h2>Reporting problems</h2>
+<p>
+    Some openskies open-source projects may adopt a project-specific code of conduct. In those cases, an Aerobridge maintainer will be identified as the Project Steward, who will receive and handle reports of code of conduct violations. 
+<br> We will investigate every complaint, but you may not receive a direct response. We will use our discretion in determining when and how to follow up on reported incidents, which may range from not taking action to permanent expulsion from the project and project-sponsored spaces. We will notify the accused of the report and provide them with an opportunity to discuss it before any action is taken. The identity of the reporter will be omitted from the details of the report supplied to the accused. In potentially harmful situations, such as ongoing harassment or threats to anyone's safety, we may take action without notice
+<p>To report a problem, write a mail to <a href="mailto:hrishi@openskies.sh">hrishi@openskies.sh</a> </p>
```

Comment:
Request to remove personal email. Instead use: https://about.openskies.sh/#contact